com.gitlab.inendi.Inspector.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: com.gitlab.inendi.Inspector
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: inspector_launcher.sh
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
+  - --filesystem=/opt/inendi
```

Comment:
Yes, that is because very large datasets are being accessed by the application and large temporary files have to be stored outside of flatpak on fast devices like RAID-0 arrays of SSDs